lib/game.rb

```diff
@@ -1,5 +1,13 @@
 # frozen_string_literal: true
 
+# Main class for Tic Tac Toe game
 class Game
-  puts 'Welcome to Tic Tac Toe!'
+  def run
+    puts 'Welcome to Tic Tac Toe!'
+    display_board
+  end
+
+  def display_board
+    puts "1 | 2 | 3\n-----------\n4 | 5 | 6\n-----------\n7 | 8 | 9"
```

Comment:
`puts` is used a second time here.

I would like to introduce a new constraint. The constraint is that you can only use `puts` once in the production code. 😄